src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder();
+        report.append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo);
+
+        for (String name: names) {
+            int salary = 0;
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/6c122fda-b3cc-4941-a850-47cbf077ed5e)

Reply:
This one was easy. Still can`t understand how I overlooked this

Reply:
I`m not sure whether the formatter should be written as a constant field (all letters are uppercase)